dicoogle/src/main/java/pt/ua/dicoogle/server/users/RolesXML.java

```diff
@@ -101,11 +101,17 @@ private String resolveAttrib( String attr, Attributes attribs, String defaultVal
     public RolesStruct getXML()
     {
         roles.reset();
-        
-        try
-        {
 
-            FileInputStream fin = new FileInputStream("roles.xml");
+        FileInputStream fin;
+
+        try {
```

Comment:
I believe one could also have a single try statement and catch `FileNotFoundException` at the end, next to the other catch blocks. But this seems fine.